packages/manager/apps/dedicated/client/app/ip/byoip/byoip.service.js

```diff
@@ -0,0 +1,58 @@
+export default class ByoipService {
+  /* @ngInject */
+  constructor($http, $q, coreConfig, User) {
+    this.$http = $http;
+    this.$q = $q;
+    this.User = User;
+    this.coreConfig = coreConfig;
+  }
+
+  getCatalog() {
+    this.user = this.coreConfig.getUser();
```

Comment:
local usage only, better like this :
```suggestion
    const user = this.coreConfig.getUser();
```